MyUnitTests/MyTargetClassTests.cs

```diff
@@ -6,23 +6,47 @@ namespace MyUnitTests
     public class MyTargetClassTests
     {
         [Fact]
-        public void VerifySummation()
+        public void Summary()
```

Comment:
Please apply this naming conventions for all your tests:
`MethodName_StateUnderTest_ExpectedBehavior`

Reply:
For example:  `Summary_WhenTwoIntegersProvided_ShouldReturnTotalAsInteger`